YesterdayApi.UserIdentity.Data/Infrastructure/Database/UserDbContext.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using YesterdayApi.UserIdentity.Data.Core.Users;
+
+namespace YesterdayApi.UserIdentity.Data.Infrastructure.Database
+{
+    public class UserDbContext : IdentityDbContext<AspIdentityUser, IdentityRole<int>, int>
+    {
+        public UserDbContext()
```

Comment:
Do we need this empty constructor? I thought one (which is below) is enough. Hmm?

Reply:
I think it's a convention or something, because i saw it in most dbcontext classes, but it doesn't give me any errors if I remove it, and i can't find the reasoning behind leaving it, so I'll remove it for now.